pkg/report/openbsd.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package report
+
+import (
+	"bytes"
+	"regexp"
+)
+
+type openbsd struct {
+	kernelSrc string
+	kernelObj string
+	ignores   []*regexp.Regexp
+}
+
+func ctorOpenbsd(kernelSrc, kernelObj string, ignores []*regexp.Regexp) (Reporter, []string, error) {
+	ctx := &openbsd{
+		kernelSrc: kernelSrc,
+		kernelObj: kernelObj,
+		ignores:   ignores,
+	}
+	return ctx, nil, nil
+}
+
+func (ctx *openbsd) ContainsCrash(output []byte) bool {
+	return containsCrash(output, openbsdOopses, ctx.ignores)
+}
+
+func (ctx *openbsd) Parse(output []byte) *Report {
+	rep := &Report{
```

Comment:
I think this whole function can now be replaced with `simpleLineParser`. See `pkg/report/gvisor.go` for an example.

Reply:
That worked, thanks!